src/background/images.ts

```diff
@@ -49,8 +50,32 @@ export async function takeScreenshot(
         uri: screenshotUri,
         blob: screenshotBlob,
       },
+      targetId: tab.id,
     })
   } catch (error) {
-    dispatchBackgroundActions.screenshotCaptureFailure({ error })
+    dispatchBackgroundActions.imageCaptureFailure({ targetId: tab.id, error })
```

Comment:
I'm not against sending the `targetId` but it doesn't look like we're using it

Reply:
Let's leave it alone, this will be useful when handling image loading